earth_enterprise/Read

```diff
@@ -0,0 +1,63 @@
+src/server/mod_fdb/portableservice.cpp:343:      reader->Read(&data_buffer, data_loc.Offset(), data_loc.Size(), r);
```

Comment:
It looks like this may have been added by mistake.

Reply:
Good catch. Not sure how I missed that.